index.html

```diff
@@ -41,11 +44,188 @@
+                        </ul>
+                        <button class="btn weight--500">See Project</button>
+                    </aside>
+                </article>
+
+                <article class="card">
+                    <div>
+                        <img src="./assets/images/Snapshoot2.svg" alt="">
+                    </div>
+                    <aside class="mt-12">
+                        <div class="weight--700 card-sub-title">
+                            Multi-Post
+                            Stories
+                        </div>
+                        <ul class="list-style-type-none d-flex weight--600 canopy justify-content-between">
+                            <li class="canopy-bold">CANOPY</li>
+                            <li class="dot"></li>
+                            <li class="">Back End Dev</li>
+                            <li class="dot"></li>
+                            <li>2015</li>
+                        </ul>
+                        <p class="card-title weight--400">A daily selection of privately personalized reads; no accounts
+                            or sign-ups required.</p>
+                        <ul class="list-style-type-none d-flex lang weight--500">
+                            <li>html</li>
+                            <li>css</li>
+                            <li>javaScript</li>
+                        </ul>
+                        <button class="btn weight--500">See Project</button>
+                    </aside>
+                </article>
             </div>
-        </header>
+        </section>
+
+        <footer>
+            <section class="container">
+                <article>
+                    <aside class="pt-114">
+                        <div class="weight--700 footer-sub-title">
+                            About
+                            Myself
+                        </div>
+                        <p class="weight--400 footer-title mt-12">
+                            Hello I’m a software developer! I can help you build a product , feature or website Look
+                            through some of my work and
+                            experience! If you like what you see and have a project you need coded, don’t hestiate
+                            to contact me.
+                        </p>
+                        <p class="nav-sub-sub-heading weight--500 text--info mt-12">
+                            LET’S CONNECT
+                        </p>
+
+                        <ul class="mt-26 d-flex list-style-type-none">
+                            <li><i class="fa-brands fa-twitter"></i></li>
+                            <li><i class="fa-brands fa-linkedin-in"></i></li>
+                            <li><i class="fa-brands fa-medium"></i></li>
+                            <li><i class="fa-brands fa-github"></i></li>
+                            <li><i class="fa-brands fa-angellist"></i></li>
+                        </ul>
```

Comment:
- [ ] Please kindly add a link `a` tag to individual icon; this will ensure that the element link the user to the appropriate page. 👍